superset/assets/src/chart/chartAction.js

```diff
@@ -83,6 +84,30 @@ export const ANNOTATION_QUERY_FAILED = 'ANNOTATION_QUERY_FAILED';
 export function annotationQueryFailed(annotation, queryResponse, key) {
   return { type: ANNOTATION_QUERY_FAILED, annotation, queryResponse, key };
 }
+export const REST_ACTION_SUCCESS = 'REST_ACTION_SUCCESS';
+export function restActionSuccess(queryResponse, key) {
+  return { type: REST_ACTION_SUCCESS, queryResponse, key };
+}
+
+export const REST_ACTION_STARTED = 'REST_ACTION_STARTED';
+export function restActionStarted(queryController,restAction, key) {
+  return { type: REST_ACTION_STARTED, restAction, queryController, key };
+}
+
+export const REST_ACTION_FAILED = 'REST_ACTION__FAILED';
```

Comment:
Seems like an extra `_` in `REST_ACTION__FAILED`

Reply:
indeed. done